packages/veritone-react-common/src/index.js

```diff
@@ -23,6 +23,9 @@ export FilePicker from './components/FilePicker';
 export ProgressDialog from './components/ProgressDialog';
 export * as formComponents from './components/formComponents';
 export video from './components/Video';
+export VideoSource from './components/Video/VideoSource';
+export MediaPlayer from './components/Video/MediaPlayer';
+export RestartMediaButton from './components/Video/RestartMediaButton';
```

Comment:
see my comment above - if this is only used specifically for the Video component then it isn't really useful to export/expose.